app/controllers/student_tasks_controller.rb

```
@@ -1,28 +1,200 @@
+      @assignment       = assignment
+      @team_participant = team_participant
+      @review_map       = review_map
+    end
+
+    def participant
+      team_participant.participant
+    end
+
+    def ensure_response!
+      map = response_map
+      return nil unless map
```

RuboCop (1.86.1)

[convention] Add empty line after guard clause.

(Layout/EmptyLineAfterGuardClause)